lib/helpers/id_validators.dart

```diff
@@ -152,6 +152,42 @@ class CustomFormFieldValidators {
     };
   }
 
+  static FormFieldValidator<T> isWalletConnectConnectionURL<T>(
+      {String? errorText, required BuildContext context}) {
+    final l10n = l10nOf(context);
+    return (T? valueCandidate) {
```

Comment:
l10n is not being used. Please localise the errors if they are going to be shown to the user

Reply:
Fixed